mlpipeline/src/mlpipeline/etl/pipeline.py

```
@@ -161,101 +148,136 @@ def process_categories(recipe_id: int, categories: list[str], cursor):
             raise err
     
     def process_ingredients(self, recipe_id: int, ingredients: list[str], cursor):
-        try:
-            docs = self.parser.process_texts(ingredients)
+        """
+        Processes raw ingredient strings and imports them into the database.
+        """
+        # TODO: Process the ingredients using the ingredient parser and then import processed ingredients.
+        for ingredient_str in ingredients:
```

Ruff (0.14.13)

Loop control variable `ingredient_str` not used within loop body

Rename unused `ingredient_str` to `_ingredient_str`

(B007)